src/llfs/no_outgoing_refs_cache.hpp

```diff
@@ -0,0 +1,95 @@
+//#=##=##=#==#=#==#===#+==#+==========+==+=+=+=+=+=++=+++=+++++=-++++=-+++++++++++
+//
+// Part of the LLFS Project, under Apache License v2.0.
+// See https://www.apache.org/licenses/LICENSE-2.0 for license information.
+// SPDX short identifier: Apache-2.0
+//
+//+++++++++++-+-+--+----- --- -- -  -  -   -
+
+#pragma once
+#ifndef LLFS_NO_OUTGOING_REFS_CACHE_HPP
+#define LLFS_NO_OUTGOING_REFS_CACHE_HPP
+
+#include <llfs/api_types.hpp>
+#include <llfs/page_id_factory.hpp>
+
+#include <batteries/bool_status.hpp>
+
+#include <atomic>
+#include <vector>
+
+namespace llfs {
+enum struct OutgoingRefsStatus : u8 {
```

Comment:
Maybe remove or at least move to the .cpp file.

Reply:
Removed as discussed below.